AWSSageMakerRuntime.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name         = 'AWSSageMakerRuntime'
+  s.version      = '2.2'
```

Comment:
We need to match this to the current SDK Version.

Reply:
Fixed